examples/mirrorinstance.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: miniooperator.min.io/v1beta1
+kind: MirrorInstance
+metadata:
+  name: minio
+## If specified, MinIOInstance pods will be dispatched by specified scheduler.
+## If not specified, the pod will be dispatched by default scheduler.
+# scheduler:
+#  name: my-custom-scheduler
+spec:
+  selector:
+    matchLabels:
+      app: mc # Should match spec.metadata.labels
+  ## Add metadata to the all pods created by the StatefulSet
+  metadata:
+    labels:
+      app: mc # Should match spec.selector.matchLabels
+  ## Registry location and Tag to download MC image
+  image: minio/mc:RELEASE.2020-04-25T00-43-23Z
+  ## This is used to set configuration
+  ## Format name: MC_HOST_<alias>
+  ## Format value: https://<access_key>:<secret_key>@<minio_server_url>
+  env:
+    - name: MC_HOST_play
+      value: "https://Q3AM3UQ867SPQQA43P2F:zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG@play.min.io"
+    - name: MC_HOST_play1
+      value: "https://Q3AM3UQ867SPQQA43P2F:zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG@play.min.io"
+  ## Use this field to set the source and destination for mirror.
+  ## source and destination can be site wide / bucket wide / prefix wide.
+  ## Please ensure to use the same aliases as set in above environment.
+  ## Format <alias>/<bucket>/<prefix>
+  args:
+    source: "play/sample-mc"
+    target: "play1/bucket2"
+    # flags:
+    #   - "watch"
```

Comment:
default to --json or --quiet - progress bar won't be displayed anyways, `watch` can become an argument bring support for other flags such as 

```
--remove
--overwrite
--multi-master (important)
--md5
--disable-multipart
--storage-class
--encrypt
--preserve
```

Reply:
Any flag that is set here, will be passed to operator and finally the `mc` command line. So we support all of the flags already

Reply:
I meant to add it here to indicate that it does support, we should show examples on how they can set with/without boolean flag, does these flags take values? if so how? 

and also implicitly add `--json` and this cannot be changed.

Reply:
Flags won't take values currently, we may need to do this a little differently. Will update the PR today

Reply:
KV support in flags is now added @harshavardhana PTAL